packages/email-sender/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "pagopa-interop-email-sender",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Pagopa Interop Email Sender",
+  "main": "dist",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "test": "vitest --config vitest.config.ts",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "tsx -r 'dotenv-flow/config' --watch  ./src/index.ts",
+    "build": "tsc && pnpm cpx './src/resources/**/*' './dist/resources'",
```

```suggestion
    "build": "tsc",
```
there's no longer a `src/resources` dir here, right?

---

I didn't notice I deleted only a portion of the command. Fixed and removed the unused dependency